config-controller/internal/controller/policy_controller.go

```diff
@@ -153,38 +182,53 @@ func (r *SecurityPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		return ctrl.Result{}, nil
 	}
 
+	policyCR.Status.Condition.UpdateCondition(configstackroxiov1alpha1.SecurityPolicyCondition{
+		Type:    configstackroxiov1alpha1.PolicyValidated,
+		Status:  "True",
+		Message: "",
+	})
+	if err := r.K8sClient.Status().Update(ctx, policyCR); err != nil {
+		errMsg := fmt.Sprintf("error updating status for securitypolicy '%s'", policyCR.GetName())
+		log.Debug(errMsg)
+		return ctrl.Result{}, errors.Wrap(err, errMsg)
+	}
+
 	// policy create or update flow
 	var retErr error
 	if desiredState.GetId() != "" {
 		log.Debugf("Updating policy %q (ID: %q)", desiredState.GetName(), desiredState.GetId())
 		if err := r.CentralClient.UpdatePolicy(ctx, desiredState); err != nil {
 			retErr = errors.Wrap(err, fmt.Sprintf("Failed to update policy '%s'", desiredState.GetName()))
-			policyCR.Status = configstackroxiov1alpha1.SecurityPolicyStatus{
-				Accepted: false,
-				Message:  retErr.Error(),
-			}
+			policyCR.Status.Condition.UpdateCondition(configstackroxiov1alpha1.SecurityPolicyCondition{
+				Type:    configstackroxiov1alpha1.AcceptedByCentral,
+				Status:  "False",
+				Message: retErr.Error(),
+			})
 		} else {
-			policyCR.Status = configstackroxiov1alpha1.SecurityPolicyStatus{
-				Accepted: true,
-				Message:  "Successfully updated policy",
-				PolicyId: desiredState.GetId(),
-			}
+			policyCR.Status.Condition.UpdateCondition(configstackroxiov1alpha1.SecurityPolicyCondition{
+				Type:    configstackroxiov1alpha1.AcceptedByCentral,
+				Status:  "True",
+				Message: "Policy was accepted by Central.",
+			})
+			policyCR.Status.PolicyId = desiredState.GetId()
 		}
 	} else {
 		log.Debugf("Creating policy with name %q", desiredState.GetName())
 		if createdPolicy, err := r.CentralClient.CreatePolicy(ctx, desiredState); err != nil {
 			retErr = errors.Wrap(err, fmt.Sprintf("Failed to create policy '%s'", desiredState.GetName()))
-			policyCR.Status = configstackroxiov1alpha1.SecurityPolicyStatus{
-				Accepted: false,
-				Message:  retErr.Error(),
-			}
+			policyCR.Status.Condition.UpdateCondition(configstackroxiov1alpha1.SecurityPolicyCondition{
+				Type:    configstackroxiov1alpha1.AcceptedByCentral,
+				Status:  "False",
+				Message: retErr.Error(),
+			})
 		} else {
 			// Create was successful so persist the policy ID received from Central
-			policyCR.Status = configstackroxiov1alpha1.SecurityPolicyStatus{
-				Accepted: true,
-				Message:  "Successfully created policy",
-				PolicyId: createdPolicy.GetId(),
-			}
+			policyCR.Status.Condition.UpdateCondition(configstackroxiov1alpha1.SecurityPolicyCondition{
+				Type:    configstackroxiov1alpha1.AcceptedByCentral,
+				Status:  "True",
+				Message: "",
```

Comment:
Would you mind going to all the success `""` messages and adding something simple?